electroncash_plugins/email_requests/qt.py

```diff
@@ -179,17 +392,21 @@ def new_invoice(self):
         #window.update_invoices_list()
 
     @hook
-    def receive_list_menu(self, menu, addr):
+    def receive_send_via_email_list_menu(self, menu, addr, uri, qrcode, status):
         window = menu.parentWidget().parent  # Grr. Electrum programmers overwrote parent() method.
```

Comment:
And update the function name & signature:

```suggestion
    def receive_list_menu_for_email_plugin(self, window, menu, addr, uri, qrcode, status):
```